test-suite/js/src/test/scala/org/scalajs/testsuite/library/BigIntTest.scala

```diff
@@ -0,0 +1,130 @@
+/*
+ * Scala.js (https://www.scala-js.org/)
+ *
+ * Copyright EPFL.
+ *
+ * Licensed under Apache License 2.0
+ * (https://www.apache.org/licenses/LICENSE-2.0).
+ *
+ * See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.
+ */
+
+package org.scalajs.testsuite.library
+
+import org.junit.Assert._
+import org.junit.Assume._
+import org.junit.{ BeforeClass, Test }
+import org.scalajs.testsuite.utils.Platform
+
+import scala.scalajs.js
+
+object BigIntTest {
+  @BeforeClass
+  def assumeRuntimeSupportsBigInt(): Unit = {
+    assumeTrue(Platform.executingInNodeJS)
+    val bigIntConstructor =
+      js.Dynamic.global.BigInt.asInstanceOf[js.UndefOr[js.Any]]
+    assumeTrue(bigIntConstructor.isDefined)
+  }
+}
+
+class BigIntTest {
+
+  // Methods we actually implement
+
+  @Test def apply(): Unit = {
+    val bi = js.BigInt("9007199254740992")
+    assertEquals(bi.toString(), "9007199254740992")
+  }
+
+  // TODO: Scala.js does not recongnize ** as operator
+  //  @Test def asIntN(): Unit = {
+  //    val x = js.BigInt(2) ** js.BigInt(64)
+  //    assertEquals(js.BigInt.asIntN(256, x), js.BigInt("18446744073709551616"))
+  //  }
```

Comment:
Looks like this is redundant wrt. to code in `operators()`.

However the name of the commented-out method is `asIntN()` but it tries to test `**`. Since I don't see another test for `asIntN()`, I think this was meant *not* to be commented, and to test `asIntN`.